libs/components/packages/collection.json

```diff
@@ -5,6 +5,11 @@
       "description": "Add @skyux/packages to your project.",
       "factory": "./src/schematics/ng-add/ng-add.schematic",
       "schema": "./src/schematics/ng-add/schema.json"
+    },
+    "update-polyfill": {
+      "factory": "./src/schematics/migrations/update-8/update-polyfill/update-polyfill.schematic#updatePolyfillSchematic",
+      "schema": "./src/schematics/migrations/update-8/update-polyfill/schema.json",
+      "description": "update-polyfill generator"
```

Comment:
This message will appear to end users when they run the schematic. What if we had something like, "Move SKY UX polyfills from test.ts/polyfills.ts to project configuration"?

Reply:
Updated.